rotkehlchen/chain/ethereum/oracles/uniswap.py

```diff
@@ -334,6 +424,7 @@ def get_pool_price(
             self,
             pool_addr: ChecksumEvmAddress,
             block_identifier: BlockIdentifier = 'latest',
+            timestamp: Timestamp | None = None,
```

Comment:
this is what I don't get completly. You can go from timestamp to block identifier. I would expect that calling this with timesatmp = None means that you query `block_identifier=latest` in the contract. It's confusing to provide both arguments. Also this function should have explanations in the docstring

Reply:
Yes, I realize this was a bit confusing and unnecessary. The block_identifier can be used for most of what I was using timestamps for. I've removed the timestamp argument from most of these functions, except for `get_price` where it is needed to raise `NoPriceForGivenTimestamp` correctly.